src/styles/_variables.scss

```diff
@@ -4,8 +4,9 @@ $heading-color: #333;
 $nav-bg-color: #FBFBFB;
 $nav-search-res-bg-color: darken($nav-bg-color, 10%);
 $nav-border-color: #CCC;
-$link-text-color: #27A0B6;
-$link-hover-color: #36D4ED;
+$link-text-color: #0062B0;
```

Comment:
This colour is not being used for/is getting overridden here:
`article h2 a, article h3 a, article h4 a, article h5 a, article h6 a`

in [main.css, line 266](https://github.com/circleci/circleci-docs/blob/master/src/styles/main.scss#L266)

![Screen Shot 2022-05-10 at 2 10 24 PM](https://user-images.githubusercontent.com/5113432/167694585-71e12226-72c4-49ec-acf1-dd9f5e92bd4c.png)

These `fa` icons are also getting text stroke applied on hover.